v1.1.1.md

```diff
@@ -0,0 +1,101 @@
+> Starting from v1.1.0, the [`install.networks`](https://docs.harvesterhci.io/v1.0/install/harvester-configuration#installnetworks) field is deprecated and replaced by the [`install.management_interface`](https://docs.harvesterhci.io/v1.1/install/harvester-configuration#installmanagement_interface) field. Please check the example file [here](https://github.com/harvester/ipxe-examples/blob/55dab7661a5c14fdf7a8da52b35e40c349f0d03c/general/config-create.yaml#L13-L20).
+ 
+
+# Highlights
+* Harvester Upgrade 1.0.3 to 1.1.0 does not handle multiple SLAVE in BOND for management interface [[#3045](https://github.com/harvester/harvester/issues/3045)]
+* Host IP is inconsistent [[#2962](https://github.com/harvester/harvester/issues/2962)]
+* Installer crash after input hostname [[#2937](https://github.com/harvester/harvester/issues/2937)]
+* GUI change: `Network` is renamed to `VM Networks` [[3088](https://github.com/harvester/harvester/issues/3088)]
+
+# Upgrade
+* [Support upgrade from v1.1.0 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+* [Support upgrade from v1.0.3 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+    * We highly recommend upgrading v1.0.3 clusters to v1.1.1 (skip version v1.1.0).
+    * Due to the implementation changes, all user VMs must shut down during an upgrade. Please stop the VMs before an upgrade.
+
+# Major Bug Fixes
+* Cant's display monitoring dashboard after upgrade, alertmanager, prometheus and grafana monitoring pods Terminating [[#2984](https://github.com/harvester/harvester/issues/2984)]
+* `migrate_harv_mgmt_to_mgmt_br.sh` should remove ClusterNetwork resource [[#3047](https://github.com/harvester/harvester/issues/3047)]
+* Need retry logic around curl in detect_repo [[#3079](https://github.com/harvester/harvester/issues/3079)]
+* Configuring the containerd registry doesn't work anymore during installation [[#3116](https://github.com/harvester/harvester/issues/3116)]
+* Tag rc version, harvester plugin needs to build the corresponding package [[#3145](https://github.com/harvester/harvester/issues/3145)]
+* Upgrade: finished upgrades interfere with newer ones [[#3152](https://github.com/harvester/harvester/issues/3152)]
+* The NICs should keep up after deleting the VLAN config [[#3166](https://github.com/harvester/harvester/issues/3166)]
+* Incomplete dependency chart installation after upgrading 1.0.3 to 1.1.0 and master-head. [[#3168](https://github.com/harvester/harvester/issues/3168)]
+* Upgrade: post-drain job fails when upgrading to the same version [[#3175](https://github.com/harvester/harvester/issues/3175)]
+* Cannot use the secret for all outputs [[#3184](https://github.com/harvester/harvester/issues/3184)]
+
+# Known Issues
+* Windows VMs crashing [[#2522](https://github.com/harvester/harvester/issues/2522)]
+* Unable to provision RKE2 cluster from external Harvester cloud credential [[#2590](https://github.com/harvester/harvester/issues/2590)] (**Deprecated**)
```

Comment:
Put this one at the last and starting with `Deprecated`.

And, have we decided to not support it in the future? It doesn't indicate in the issue and still marked for v1.1.2.

Reply:
Yes, and we have planned to remove the UI in v1.1.2 completely.